crates/flox-rust-sdk/src/models/environment/path_environment.rs

```diff
@@ -43,7 +43,7 @@ const ENVIRONMENT_DIR_NAME: &'_ str = "env";
 /// The transaction status is captured via the `state` field.
 #[derive(Debug)]
 pub struct PathEnvironment<S> {
-    /// Absolute path to the environment, typically within `<...>/.flox/name`
+    /// Absolute path to the environment, typically `<...>/.flox/env`
```

Comment:
suggestion nonblocking: store `<...>/.flox` and avoid calling `parent()` in `out_link`

Reply:
I'd prefer that too

Reply:
Done